### DISCUSSION TITLE
Run more than one instance, more tray menus

### DISCUSSION THREAD

**Discussion by fatehiman** [Hofknecht/SystemTrayMenu#535] [Ideas]
Thank you for your excellent app!
I recently upgraded from Windows 10 to Windows 11. Previously, I had four toolbars set up, and I’d like to replicate that setup using your app. However, when I attempt to run multiple instances with different settings (by copying the app into separate folders), launching a second instance causes the first to close.
Could you please advise on how I can run multiple instances simultaneously? Additionally, it would be fantastic if future versions of the app could support multiple root folders within the same instance with separate tray icons for each root folder.
Thank you for your time and consideration!

**Comment by kdmoyers** [comment]
Perhaps two things are needed?
- SystemTrayMenu.exe takes a command line argument that is the folder where a config file is located
- SystemTrayMenu.exe permits multiple running instances, each with it's own config file

**Reply by Gitoffthelawn** [reply]
>     * SystemTrayMenu.exe permits multiple running instances, each with it's own config file

Would it be more efficient to run a single instance that supports multiple system tray icons?  I think I would prefer such functionality.

**Reply by tbstauffer** [reply]
i would also vote for this. one instance, multiple directories. I see that the app supports multiple directories, which is cool, but there is no way to distinguish which shortcut is from which directory.  In my case, all the directories have the same shortcuts but from a different location. So having the names show up 2 or 3x in the same menu is confusing
